Course.php

```diff
@@ -236,6 +237,7 @@
        </div>
    <div id="menu3" class="container tab-pane"><br>
          <?php
+         $group_id=$_SESSION['group_id'];
```

Comment:
This line appears on line 172 too.   Does the duplication matter?

**Hui**

Reply:
the variable was losing value, maybe it wasn't global will check it out.